cmd/set/backend.go

```diff
@@ -0,0 +1,46 @@
+package set
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/puppetlabs/prm/pkg/prm"
+	"github.com/spf13/cobra"
+)
+
+var SelectedBackend prm.BackendType
+
+func (sc *SetCommand) createSetBackendCommand() *cobra.Command {
+	tmp := &cobra.Command{
+		Use:     "backend <BACKEND>",
+		Short:   "Sets the backend exec environment to the specified type",
+		Long:    `Sets the backend exec environment to the specified type`,
+		PreRunE: sc.setBackendPreRunE,
+		RunE:    sc.setBackendType,
```

Comment:
```suggestion
		RunE:    sc.setBackendType,
		ValidArgs: []string{string(prm.DOCKER)},
```

Also add to the help text the list of valid options.